frontend/src/locales/zh-CN.yml

```diff
@@ -95,7 +135,7 @@ settings:
     plan: （升级到团队方案来开启角色管理）
   workspace:
     url-section: SQL Console URL
-    url-tip: '提示: 用 {schema} 作为真实数据库名称的占位符'
+    url-tip: "提示: 用 {schema} 作为真实数据库名称的占位符"
```

Comment:
I noticed this is changed back and force among different committers, can we define a rule here, whether we should use quote, and if used, when to use single or double quote?

@yaohui-wyh @ecmadao